ogcore/TPI.py

```diff
@@ -704,7 +704,7 @@ def run_TPI(p, client=None):
     total_tax_revenue = np.ones(p.T + p.S) * ss_vars["total_tax_revenue"]
 
     # Compute other interest rates
-    r_gov = fiscal.get_r_gov(r, p, "TPI")
+    r_gov = fiscal.get_r_gov(r, 0, p, "TPI")
```

Comment:
@jdebacker. Why is the `DY_ratio` argument in this line equal to 0? You have the `D` and the `Y` series above.

Reply:
I've updated this to take the initial guess at the debt to GDP path (although since these are initial guesses, they shouldn't affect the model solution, but make affect the time to convergence).